packages/server/test/integration/http_requests_spec.js

```diff
@@ -344,12 +343,13 @@ describe('Routes', () => {
         .then((res) => {
           expect(res.statusCode).to.eq(200)
 
-          expect(res.body).to.match(/Runner.start\(.+\)/)
+          expect(res.body).to.match(/window.__Cypress__ = true/)
         })
       })
     })
 
-    it('clientRoute routes to \'not launched through Cypress\' without a proxy set', function () {
+    // TODO: no automation in unified app
+    it.skip('clientRoute routes to \'not launched through Cypress\' without a proxy set', function () {
```

Comment:
Do we need to come back to this? If so should we temp skip or add a ticket so we remember to?

Reply:
I think this is fixed, I’ll try and unskip

Reply:
Edit, yep it's good, unskipped